deploy/charts/openebs-monitoring/Chart.yaml

```diff
@@ -46,3 +46,7 @@ dependencies:
     version: "16.5.*"
     repository: https://prometheus-community.github.io/helm-charts
     condition: kube-prometheus-stack.install
+  - name: node-problem-detector
+    version: 2.0.2
```

Comment:
Use wildcards for specifying version (`version: "2.0.*" `)?
Helm chart best practices: [versions](https://helm.sh/docs/chart_best_practices/dependencies/#versions)